modules/distr-tracing-tempo-install-instance.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+//* distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-tempo-install-instance_{context}"]
+= Installing a {TempoName} instance
+
+https://issues.redhat.com/browse/TRACING-3296
+
+Deploying the TempoStack
+
+You can use the OpenShift Container Platform web console to deploy the TempoStack.
+
+.Prerequisites
+
+* Supported Object Store (AWS S3, Google Cloud Storage, Azure, Swift, Minio, OpenShift Data Foundation)
+
+.Procedure
+
+. Install the Tempo Operator:
+
+.. In the OpenShift Container Platform web console, click *Operators* → *OperatorHub*.
+
+.. Choose *Tempo Operator* from the list of available Operators, and click *Install*.
+
+.. Under *Installation Mode*, select *All namespaces* on the cluster.
+
+.. Under *Installed Namespace*, select *openshift-operators-redhat*.
++
+IMPORTANT: You must specify the `openshift-operators-redhat` namespace. The `openshift-operators` namespace might contain Community Operators, which are untrusted and might publish a metric with the same name as an OpenShift Container Platform metric, which would cause conflicts.
+
+. Select an Approval Strategy:
+** The Automatic strategy allows Operator Lifecycle Manager (OLM) to automatically update the Operator when a new version is available.
+** The Manual strategy requires a user with appropriate credentials to approve the Operator update.
+
+. Click *Install*.
+
+. Verify that you installed the Tempo Operator:
+..  Go to *Operators* → *Installed Operators* page and look for *Tempo Operator*.
+.. Ensure that *Tempo Operator* is listed with *Status* as *Succeeded* in all the projects.
+
+. Create a Secret YAML file that uses the `access_key_id` and `access_key_secret` fields to specify your AWS credentials and bucketnames, endpoint and region to define the object storage location. For example:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: minio-test
+stringData:
+  endpoint: http://minio.minio.svc:9000
+  bucket: tempo
+  access_key_id: tempo
+  access_key_secret: supersecret
+type: Opaque
+----
```

Comment:
I'm not sure if we should do it for AWS S3 or ODF.

Reply:
Its fine if we use S3, many of the operators use S3 as an example. However the name should be different for the secret and we can use the following example to create the secret. 

```
oc create secret generic tempostack-dev-s3 \
  --from-literal=bucket="<BUCKET_NAME>" \
  --from-literal=endpoint="<AWS_BUCKET_ENDPOINT>" \
  --from-literal=access_key_id="<AWS_ACCESS_KEY_ID>" \
  --from-literal=access_key_secret="<AWS_ACCESS_KEY_SECRET>"
```

Reply:
Outdated.Resolving this thread.